core/src/main/scala/kafka/controller/ControllerChannelManager.scala

```diff
@@ -368,7 +392,7 @@ class RequestSendThread(val controllerId: Int,
         val requestHeader = clientResponse.requestHeader
         val api = requestHeader.apiKey
         if (api != ApiKeys.LEADER_AND_ISR && api != ApiKeys.STOP_REPLICA && api != ApiKeys.UPDATE_METADATA &&
-          api != ApiKeys.LI_COMBINED_CONTROL)
```

Comment:
Maybe pull out this ApiKeys to a declared constant Set so that 
1. We can use set.contains to evaluate here
2. The above foreach can loop on the same ApiKeys set

Reply:
Done